code/mob/melee_attack_procs.dm

```diff
@@ -479,7 +479,7 @@
 #undef DISARM_WITH_ITEM_TEXT
 
 /mob/proc/check_block(ignoreStuns = 0) //am i blocking?
-	if (ignoreStuns || (isalive(src) && !getStatusDuration("paralysis")))
+	if (ignoreStuns || can_act(src, 0))
```

Comment:
mechanical change - this was specifically changed in past from stun+weak+para check to just para

Reply:
revert